napalm/base/helpers.py

```diff
@@ -19,6 +20,11 @@
 from napalm.base.utils.jinja_filters import CustomJinjaFilters
 from napalm.base.canonical_map import base_interfaces, reverse_mapping
 
+# ----------------------------------------------------------------------------------------------------------------------
```

Comment:
I'm surprised pylama didn't complain about this long line :thinking:

Reply:
Syntax may be ignored on comments but we can trim it down to PEP8's 79 char line limit

Reply:
This was resolved  in commit [
2130fcc](https://github.com/napalm-automation/napalm/pull/1132/commits/2130fcc1c245c28b423b481a8d019dc5e99395d2) so we should be OK now